atlp-rwanda/e-commerce-lydia-32-fn
# Ft test setup circleci config

### What does this PR do?
This Pull Request serves to setup testing environment for our components

### Description of Task to be completed?
Setting Jest testing environment.
Running some sample tests to see if they can work
###How should this be manually tested?
Inside root directory run **yarn test** and **yarn test:coverage** if you want to see coverage
